scripts/presubmit.sh

```diff
@@ -128,8 +132,14 @@ main() {
           ${goflags} "$d"
     done | xargs -I '{}' -P ${GO_TEST_PARALLELISM:-10} bash -c '{}'
 
-    [[ ${coverage} -eq 1 ]] && \
-      cat /tmp/trillian_profile/*.out > /tmp/coverage.txt
+    cat /tmp/trillian_profile/*.out > /tmp/coverage.txt
+  fi
+  else
+      go test \
+        -short \
+        -timeout=${GO_TEST_TIMEOUT:-5m} \
+        ${coverflags} \
```

Comment:
Could remove `${coverflags}`, since it'll always be empty.

Reply:
done